axelor-account/src/main/java/com/axelor/apps/account/service/moveline/MoveLineConsolidateServiceImpl.java

```diff
@@ -116,18 +117,28 @@ public List<MoveLine> consolidateMoveLines(List<MoveLine> moveLines) {
 
     Map<List<Object>, MoveLine> map = new HashMap<List<Object>, MoveLine>();
     MoveLine consolidateMoveLine = null;
+    boolean haveOldBack =
+        moveLines.stream()
+            .anyMatch(
+                ml ->
+                    ml.getMove().getPaymentCondition().getPaymentConditionLineList().stream()
```

Comment:
paymentCondition can be null on a move, please improve this statement to not have NPE